tabs/thredds_frame_source.py

```diff
@@ -189,7 +191,7 @@ def main(NFRAMES=90, output_dir=None):
         output_dir = os.path.join(os.path.dirname(__file__),
                                   '../static/data/json')
     filename = partial(os.path.join, output_dir)
-    frame_source = THREDDSFrameSource(DEFAULT_DATA_URI, decimate_factor=10)
+    frame_source = THREDDSFrameSource(HINDCAST_CACHE_DATA_URI, decimate_factor=10)
```

Comment:
I's always passed explicitly in the thredds connection, so this only applies to the standalone test of the connection. I don't think it matters much here.